ui/main/src/app/services/map.service.ts

```diff
@@ -0,0 +1,21 @@
+import {EventEmitter, Injectable} from '@angular/core';
+import {LightCard} from '@ofModel/light-card.model';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class MapService {
+    highlightCardEvent = new EventEmitter<any>();
+    zoomToLocationEvent = new EventEmitter<string>();
+
+    constructor() {}
+
+    highlightOnMap(highLight: boolean, lightCard: LightCard) {
+        const lightCardId = lightCard.id;
+        this.highlightCardEvent.emit({lightCardId, highLight});
```

Comment:
Doesn't work, unfortunately. Because here I emit an object with two properties (lightCardId and highLight) and properties cannot have a dot (.) in their name.

Reply:
ok